### PR TITLE
TICK-0002 created Product DTO.

### PR DESCRIPTION
**Pull request by DanMedl**
//I was thinking about unique id and try to use class from Java.Lang to generate it automatically. Because of using UUID, we don't need setter for this field because it's generating automatically. I am gonna be waiting for you feedback.

### CODE REVIEW

File: com.atb/src/main/java/com/atb/dtos/Product.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import java.util.UUID;
+
+public class Product {
+    private static int id;
+    private String code;
+    private String description;
+    private double price;
+
+    public Product( String code, String description, double price) {
```
Comment (Shaff3004) [inline com.atb/src/main/java/com/atb/dtos/Product.java:11]: change this constructor to default one (without parameters)

File: com.atb/src/main/java/com/atb/dtos/Product.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package com.atb.dtos;
+
+import java.util.UUID;
```
Comment (Shaff3004) [inline com.atb/src/main/java/com/atb/dtos/Product.java:3]: (minor) remove this import

File: com.atb/src/main/java/com/atb/dtos/Product.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+public class Product {
+    private static int id;
```
Comment (Shaff3004) [inline com.atb/src/main/java/com/atb/dtos/Product.java:6]: (minor) add empty line between class name and variables definition (it relates to all classes)

File: com.atb/src/main/java/com/atb/dtos/Product.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        this.code = code;
+        this.description = description;
+        this.price = price;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+
```
Comment (Shaff3004) [inline com.atb/src/main/java/com/atb/dtos/Product.java:21]: (minor) remove empty line (only one line should be used as a separator between methods

File: com.atb/src/main/java/com/atb/dtos/Product.java (hunk 1 above)
Comment (DanMedl) [reply com.atb/src/main/java/com/atb/dtos/Product.java:11]: done

File: com.atb/src/main/java/com/atb/dtos/Product.java (hunk 2 above)
Comment (DanMedl) [reply com.atb/src/main/java/com/atb/dtos/Product.java:3]: done

File: com.atb/src/main/java/com/atb/dtos/Product.java (hunk 3 above)
Comment (DanMedl) [reply com.atb/src/main/java/com/atb/dtos/Product.java:6]: done

File: com.atb/src/main/java/com/atb/dtos/Product.java (hunk 4 above)
Comment (DanMedl) [reply com.atb/src/main/java/com/atb/dtos/Product.java]: done

**Pull request merged**